lisp/ess-custom.el

```diff
@@ -2321,41 +2325,88 @@ and `ess-tracebug'.")
 
 ;;*;; Inferior ESS commands
 
-(defvar ess-load-command "source(\"%s\")\n"
+(defvar ess-load-command "source('%s')\n"
   "Dialect specific format-string for building the ess command to load a file.
 
 This format string should use %s to substitute a file name and should
 result in an ESS expression that will command the inferior ESS to load
 that file.")
-(define-obsolete-variable-alias 'inferior-ess-load-command 'ess-load-command "ESS v13.09")
-
-(defvar ess-load-visibly-command nil
```

Comment:
> ESS always dealt with two types of evaluation - visual and non-visual.

Nothing has changed on that point. `visibly` gets treated at two levels: in the evaluation functions themselves (interaction with inferior), and within the inferior. We are only talking about the latter.

The R dialect had special command variables with different combinations of `visibly` and `noecho`. I needed to add `package` and `verbose` parameters for namespaced evaluation. Are we to multiply exponentially the number of evaluation commands? Both for eval and load? We need a function here.

The signature for the eval formatting function is

```lisp
(defun ess-format-eval-command (string &optional visibly output file &rest args))
```

and for load it's

```lisp
(defun ess-format-load-command (file &optional visibly output &rest args))
```

The `&rest args` is meant for dialect specific parameters. All other parameters are ESS-wide. I think that is a reasonable scheme.

Maybe we need to move `output` to `&rest args` since it's currently R specific though.